src/input.js

```diff
@@ -40,4 +40,19 @@ export default class Input extends Plugin {
 		injectUnsafeKeystrokesHandling( editor );
 		injectTypingMutationsHandling( editor );
 	}
+
+	/**
+	 * Checks batch if it is a result of user input - e.g. typing.
+	 *
+	 * **Note:** This method checks if the batch was created using {@link module:typing/inputcommand~InputCommand 'input'}
+	 * command as typing changes coming from user input are inserted to the document using that command.
+	 *
```

Comment:
I think it would be nice to provide an example here. Check the issue discussion -- I've posted an example snippet there.

Reply:
It doesn't cost us much and if anyone would wonder how to exactly use it then they will have a snippet.

Reply:
I've added a really simple one - probably is enough.